applications/serial_lte_modem/src/gnss/Kconfig

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+config SLM_GNSS
+	bool "GPS support in SLM"
+	default y
+
+config SLM_GPS
+	bool "GPS support in SLM"
+	default y if SLM_GNSS
+	select NRF9160_GPS
```

Comment:
Why there are both SLM_GNSS and SLM_GPS configuration points? So that the application configuration doesn't have to be changed?

Reply:
Will add SLM_CELL_POS in which case SLM_GPS is not needed